src/scripts.js

```diff
@@ -10,204 +10,145 @@
 // import User from './user';
 // import Cookbook from './cookbook';
 // import domUpdates from './domUpdates';
-
+let favorites = []
```

Comment:
I'm now wondering if this needed to be global and that's why you created a new variable for `favorites`... is there a way we can use the `favoriteRecipes` array on the User object while making those fetch requests?  Is that something you tried initially and found this was a better way?